docs/sql-data-sources-avro.md

```diff
@@ -240,6 +240,14 @@ Data source options of Avro can be set via:
     </td>
     <td>function <code>from_avro</code></td>
   </tr>
+  <tr>
+    <td><code>writerSchema</code></td>
```

Comment:
How about `actualSchema`? I think it is more straightforward.

Reply:
I would stick to `writerSchema`, mostly because this is also the term used in Avro itself: https://avro.apache.org/docs/1.9.1/api/java/org/apache/avro/hadoop/io/AvroValueDeserializer.html

Reply:
I come up with this because in the implementation it also uses the name `actual`: https://github.com/rdblue/avro-java/blob/master/avro/src/main/java/org/apache/avro/generic/GenericDatumReader.java#L67

But I am OK with `writerSchema` as well since it is the name in the constructor method.

Reply:
Yeah, actually `writerSchema` name is super confusing to me. Can we use `actualSchema`? I would prefer this one more.